iOSClient/Data/NCDatabase.swift

```diff
@@ -109,24 +109,6 @@ class tableGPS: Object {
     @objc dynamic var placemarkThoroughfare = ""
 }
 
-class tableLocalFile: Object {
-
-    @objc dynamic var account = ""
-    @objc dynamic var etag = ""
-    @objc dynamic var exifDate: NSDate?
-    @objc dynamic var exifLatitude = ""
-    @objc dynamic var exifLongitude = ""
-    @objc dynamic var exifLensModel: String?
-    @objc dynamic var favorite: Bool = false
-    @objc dynamic var fileName = ""
-    @objc dynamic var ocId = ""
-    @objc dynamic var offline: Bool = false
-
-    override static func primaryKey() -> String {
-        return "ocId"
-    }
-}
-
```

Comment:
I removed the exif fields on my PR here https://github.com/nextcloud/ios/pull/2556/files#diff-a5a2c204e96700f71958b1d97b10295b8b331670b25d51b397f674ed291d2ee5L115-R115 , but it seems like you removed this class all-together. I may need some help on this moving forward.